### PR TITLE
Update map-editor.md

### PR DESCRIPTION
**Pull request by Not-Noah16**
# Beginner's Guide to Modd.io Map Editor

**Modd.io** is a platform that allows you to create and share multiplayer online games. The map editor is a tool for designing the game world. In this guide, we'll walk you through the basics of using the map editor, including its main function features.

## Understanding the Interface

1. The map editor interface consists of various panels and tools. Familiarize yourself with these elements:
   - **Canvas**: This is the central area where you'll design your map.
   - **Toolbar**: Located at the top, it contains various tools for map editing.
   - **Properties Panel**: On the right, it displays information about selected objects or tiles.
   - **Tile Palette**: On the Bottom right, it provides a selection of tiles to add to your map.

## Navigating the Map Editor Tools

The map editor offers several tools that are essential for map creation:

### Cursor Tool (Press C for shortcut use)

- Use the Cursor Tool to interact with regions and entities on the map.
- Select this tool by pressing "**C**" or clicking on its icon in the toolbar.

### Entities Tool (Press A for shortcut use)

- The Entities Tool allows you to place selected entities on the map.
- Select this tool by pressing "**A**" or clicking on its icon in the toolbar.

### Stamp Brush Tool (Press B for shortcut use)

- The Stamp Brush Tool has two functions:
   - Left Mouse Button (**LMB**): Place selected tiles on the map.
   - Right Mouse Button (**RMB**): Copy tiles from one location to another.
- Select this tool by pressing "**B**" or clicking on its icon in the toolbar.

### Eraser Tool (Press E for shortcut use)

- Use the Eraser Tool to delete tiles from the selected area.
- Select this tool by pressing "**E**" or clicking on its icon in the toolbar.

### Bucket Fill Tool (Press F for shortcut use)

- The Bucket Fill Tool allows you to fill an area with the selected tile.
- Select this tool by pressing "**F**" or clicking on its icon in the toolbar.

### Clear Layer (Press L for shortcut use)

- The Clear Layer Tool clears the selected layer of the map.
- Select this tool by pressing "**L**" or clicking on its icon in the toolbar.

### Region Tool (Press R for shortcut use)

- Use the Region Tool to draw a new region on the map.
- Select this tool by pressing "**R**" or clicking on its icon in the toolbar.

### Save Map Tool (Press S for shortcut use)

- The Save Map Tool is used to save all changes made to the map.
- Select this tool by pressing "**S**" or clicking on its icon in the toolbar.

### Undo (Press CTRL + Z for shortcut use)

- The Undo feature allows you to revert your last action in case of mistakes or changes.

### Redo (Press CTRL + SHIFT + Z Or CTRL + Y for shortcut use)

- The Redo feature lets you reverse an Undo action if needed.

## Working with Map Layers

The **modd.io map editor** allows you to organize your map into layers, each serving a specific purpose:

### Trees (The user is able to walk through and under)

- The Trees layer is used for objects such as trees, bushes, or any elements that players can walk through and under.
- Objects on this layer should not block the player's movement.
- Use the Stamp Brush Tool to place objects on this layer.

### Walls (The user is unable to walk through)

- The Walls layer is for solid barriers or obstacles that block the player's movement.
- Walls should prevent players from passing through them.
- Use the Stamp Brush Tool to place objects on this layer.

### Floor 2 (This layer is visible on top of the floor layer)

- The Floor 2 layer can be used for adding decorative elements or objects that appear on top of the main floor layer.
- These objects might include decorations, furniture, or items that enhance the game's visual appeal.
- Use the Stamp Brush Tool to place objects on this layer.

### Floor (The main layer)

- The Floor layer is the primary layer where you'll design the main terrain, ground, or environment.
- It serves as the foundation for your map.
- Use the Stamp Brush Tool or Paint Bucket Fill Tool to paint the main floor tiles.

## Other Main Functions

- To access map settings and customize your map further, go to the "Game Settings" menu and select "Map."
- Here, you'll have visibility into additional options:

### Change Map Size

- Adjust the map size to meet the requirements of your game.

### Change and Download the Tile Sheet (JSON)

- Modify and download the tile sheet in JSON format to fine-tune your map's tileset.

## Play Your Map

- Click on the "**Play**" button on the top panel to playtest your map.
- Make any necessary adjustments based on your gameplay experience.

*Guide by Not_Noah*

### CODE REVIEW

File: workspaces/map-editor.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Map Editor
-
-*This page needs help. Find out how to contribute [here](../../README.md)*
\ No newline at end of file
+k
```
Comment (sloont) [inline workspaces/map-editor.md:2]: could you add what you posted in the github comment to the `.md` file instead of just the letter `k`?

**Review by sloont — CHANGES_REQUESTED** [review]

File: workspaces/map-editor.md (hunk 1 above)
Comment (Not-Noah16) [reply workspaces/map-editor.md:2]: Could you tell me how

Comment (Not-Noah16) [reply workspaces/map-editor.md:2]: I'm not sure if I did it right, but I copied what I already wrote and went to https://github.com/moddio/moddio-docs/blob/main/workspaces/map-editor.md . Then, I clicked on "Edit in Place" and pasted the content

Comment (sloont) [reply workspaces/map-editor.md:2]: i think you have to edit your own branch. https://github.com/moddio/moddio-docs/blob/96504648786e8525fd78093c013728af042f55c4/workspaces/map-editor.md this should be the link to it

Comment (Not-Noah16) [reply workspaces/map-editor.md:2]: Ok, I think I've done it

Comment (sloont) [reply workspaces/map-editor.md:2]: its not on this PR which means its not on this branch. https://github.com/Not-Noah16/moddio-docs/commits/patch-3 

you can make a new PR with the branch where you made the changes, or you can add them to this branch

Comment (Not-Noah16) [reply workspaces/map-editor.md:2]: I think i made a new PR, if not, I'm not sure how to

**Comment by sloont** [comment]
continued in #11

**Pull request closed without merging**